studio/config/docker/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN apt-get --allow-releaseinfo-change update && \
 # install conda & packages
 RUN mkdir -p /opt/miniconda3 && \
     apt-get install --no-install-recommends -y gcc g++ libgl1 libgl1-mesa-dev libopencv-dev && \
+    apt-get install -y curl && \
```

Comment:
If there is no particular reason, the `apt-get install` command should be merged with the command on the line above, instead of being split into multiple steps.